src/OrchardCore.Modules/OrchardCore.AuditTrail/Drivers/AuditTrailPartSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OrchardCore.AuditTrail.Models;
+using OrchardCore.AuditTrail.Settings;
+using OrchardCore.AuditTrail.ViewModels;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.ContentTypes.Editors;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using System.Threading.Tasks;
+
+namespace OrchardCore.AuditTrail.Drivers
+{
+    public class AuditTrailPartSettingsDisplayDriver : ContentTypePartDefinitionDisplayDriver
+    {
+        public override IDisplayResult Edit(ContentTypePartDefinition contentTypePartDefinition, IUpdateModel updater)
+        {
+            if (!string.Equals(nameof(AuditTrailPart), contentTypePartDefinition.PartDefinition.Name)) return null;
+
+            return Initialize<AuditTrailPartSettingsViewModel>($"{nameof(AuditTrailPartSettings)}_Edit", model =>
```

Comment:
Don't think we need the extra string interpolation here, I think it prevents the compiler optimizing these